isce2_topsapp/packaging_utils/additional_layers.json

```diff
@@ -6,6 +6,15 @@
                                        "long_name": "ionospherePhaseCorrection",
                                        "description": "Estimated ionosphere phase correction"
                                        }
-                             }
+                             },
+ "ionosphereBurstRamps": {"dst_group": "/science/grids/corrections/derived/ionosphereBurstRamps",
```

Comment:
added to separate dst_group due to different resolution between ionosphere (downsampled) and ionosphereBurstRamps (full-res)